docs/rules/class-methods-use-this.md

```diff
@@ -83,3 +83,35 @@ class A {
     }
 }
 ```
+
+## Options
+
+### Exceptions
+
+```
+"class-methods-use-this": [<enabled>, { "exceptions": [<...exceptions>] }]
+```
+
+The `exceptions` option allows you to pass an array of method names for which you'd like to ignore warnings.
```

Comment:
Am I totally bonkers if I would like to see "you would like" instead of "you'd like"?